cmd/kubeadm/app/phases/addons/proxy/proxy_test.go

```diff
@@ -107,14 +111,15 @@ func TestCompileManifests(t *testing.T) {
 		expected bool
 	}{
 		{
-			manifest: KubeProxyConfigMap,
-			data: struct{ MasterEndpoint string }{
+			manifest: KubeProxyConfigMap19,
+			data: struct{ MasterEndpoint, ProxyConfig string }{
 				MasterEndpoint: "foo",
+				ProxyConfig:    "0.0.0.0",
 			},
 			expected: true,
 		},
 		{
-			manifest: KubeProxyDaemonSet,
+			manifest: KubeProxyDaemonSet19,
```

Comment:
let's not lose v18 coverage, add coverage for the v1.8 variants here as well

Reply:
done